mcs/class/System.Net.Http/System.Net.Http/HttpClient.wasm.cs

```diff
@@ -6,49 +6,46 @@ namespace System.Net.Http
 {
     public partial class HttpClient
     {
-
-#pragma warning disable 649
-        private static Func<HttpMessageHandler> GetHttpMessageHandler;
-#pragma warning restore 649
-
         internal static HttpMessageHandler CreateDefaultHandler()
         {
 
-            if (GetHttpMessageHandler == null)
+            string envvar = Environment.GetEnvironmentVariable ("WASM_HTTP_CLIENT_HANDLER_TYPE")?.Trim ();
```

Comment:
Do we really need this?

Reply:
It is something that @lewing and I were discussing on an alternate way to remove the static method.  

There is also this PR: https://github.com/mono/mono/pull/15051 that is more the XI style but it requires Blazor and others to buy into our full bindings stack.